core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -6,44 +6,83 @@
+import com.badlogic.gdx.graphics.g2d.*;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.actions.Actions;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
 import de.adesso.brainysnake.Gamelogic.Entities.GameObject;
 import de.adesso.brainysnake.Gamelogic.Game;
+import de.adesso.brainysnake.Gamelogic.GameMaster;
 import de.adesso.brainysnake.Gamelogic.IO.KeyBoardControl;
+import de.adesso.brainysnake.Gamelogic.Player.PlayerController;
+import de.adesso.brainysnake.Gamelogic.Player.PlayerHandler;
 import de.adesso.brainysnake.Gamelogic.UI.UIPlayerInformation;
 import de.adesso.brainysnake.Gamelogic.UI.UiState;
 import de.adesso.brainysnake.playercommon.math.Point2D;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class BrainySnake extends ApplicationAdapter {
 
     private Texture texture;
+    private Texture logoTexture;
+    private Image logoBrainySnake;
     private SpriteBatch gameSpriteBatch;
     private SpriteBatch fontSpriteBatch;
     private Sprite sprite;
     private Pixmap pixmap;
 
-    private static int DOT_SITZE = 10;
-    private static int WIDTH = Config.APPLICATION_WIDTH / DOT_SITZE;
-    private static int HEIGHT = Config.APPLICATION_HEIGHT / DOT_SITZE;
+    private Stage stage;
+    private Skin skin;
+
+    private GameMaster gameMaster;
+
+    private final int NAME_OFFSET = 75;
+    private static int DOT_SIZE = 10;
+    private static int WIDTH = Config.APPLICATION_WIDTH / DOT_SIZE;
+    private static int HEIGHT = Config.APPLICATION_HEIGHT / DOT_SIZE;
     private static int APPLICATION_WIDTH = Config.APPLICATION_WIDTH;
     private static int APPLICATION_HEIGHT = Config.APPLICATION_HEIGHT;
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(PlayerController.class.getName());
+
     private OrthographicCamera mainCamera;
     private OrthographicCamera fontCamera;
 
+    private TextButton returnButton;
+    private TextButton newGameButton;
+    private TextButton startGameButton;
+    private TextButton exitButton;
+
     private static final float MIN_FRAME_LENGTH = 1f / Config.UPDATE_RATE;
     private float timeSinceLastRender = 0;
 
     private BitmapFont font;
     private List<GameObject> gameObjects;
     private Game game;
-    private boolean gameStarted;
+    private boolean menuShowing = true;
+    private boolean matchMenuShowing = false;
 
     @Override
     public void create() {
         pixmap = new Pixmap(WIDTH, HEIGHT, Pixmap.Format.RGBA8888);
+        stage = new Stage();
+        createBasicSkin();
+
+        try {
+            logoTexture = new Texture(Gdx.files.internal("./core/src/de/adesso/brainysnake/img/BrainySnake_Headline.png"));
```

Comment:
Ich glaube das hatten wir schon?
Bitte in den assets ordner

Reply:
Im aktuellen Branch ist das normalerweise schon bereits behoben. Basiert der Kommentar vielleicht auf einem älteren Commit?

Reply:
@FrederikSchlemmer Die Requests beziehen sich immer auf den Stand, der zum Zeitpunkt des Reviews auf dem Branch herrscht. Wenn auf dem PR noch weitere Pushes folgen, kommt es zu obigen Verwirrungen, was ich aber nicht weiter wild finde. Evtl. müssen wir da noch etwas justieren.